scripts/bert/finetune_classifier.py

```diff
@@ -135,6 +134,19 @@
     'for both bert_24_1024_16 and bert_12_768_12.'
     '\'wiki_cn\', \'wiki_multilingual\' and \'wiki_multilingual_cased\' for bert_12_768_12 only.'
 )
+parser.add_argument(
```

Comment:
There are potentially two use cases for checkpointing:
- further pre-train a bert model (say, the one provided by google) on a custom dataset using gluonnlp pre-training script, save a checkpoint, and then load it for fine-tuning classification/regression task. 
- run fine-tuning classification/regression for a few epochs, save a checkpoint, then resume fine-tuning from the checkpoint. 

Both are valid use case. However, for the second use case, we need to load both model parameter and trainer state (the mean and var in Adam) to resume training. 

I suggest we support the first use case here and explicitly document what the model parameters are for. For the second use case, usually fine-tuning is less expensive, there is less need to checkpoint and resume.

To support the first use case, we need to change the code to model.bert.load_parameters

Reply:
@eric-haibin-lin OK，I got it. In this case we need to add a command parameter to provide the parameters of the pre-trained bert model. But if the bert model is trained on custom data, will BERTVocab change? Does this have an impact?
If the BERTVocab has changed due to the use of custom data, then just loading the parameters may not be appropriate.

Reply:
The BERTVocab from the get_bert_model api should work fine. The vocab from google is usually good enough